keyboards/ergodox_infinity/ergodox_infinity.c

```diff
@@ -117,12 +128,30 @@ void matrix_init_kb(void) {
 #ifndef VISUALIZER_ENABLE
 	lcd_backlight_hal_init();
 #endif
+#ifdef WPM_ENABLE
+	add_remote_objects(remote_objects, sizeof(remote_objects) / sizeof(remote_object_t*));
+#endif
 }
 
 void matrix_scan_kb(void) {
 	// put your looping keyboard code here
 	// runs every cycle (a lot)
 
+#ifdef WPM_ENABLE
+	if (is_serial_link_master()) {
+		uint8_t current_wpm = get_current_wpm();
+		if (current_wpm != last_sent_wpm) {
+			*begin_write_current_wpm() = current_wpm;
+			end_write_current_wpm();
+			last_sent_wpm = current_wpm;
+		}
+	} else if (is_serial_link_connected()) {
+		uint8_t* new_wpm = read_current_wpm();
+		if (new_wpm) {
+			set_current_wpm(*new_wpm);
+		}
+	}
```

Comment:
```suggestion
    if (is_serial_link_master()) {
        uint8_t current_wpm = get_current_wpm();
        if (current_wpm != last_sent_wpm) {
            *begin_write_current_wpm() = current_wpm;
            end_write_current_wpm();
            last_sent_wpm = current_wpm;
        }
    } else if (is_serial_link_connected()) {
        uint8_t* new_wpm = read_current_wpm();
        if (new_wpm) {
            set_current_wpm(*new_wpm);
        }
    }
```

Reply:
Yeah, the entire file was a bit of a mixed bag on that front, I tried to match the indentation of the lines close to the ones I changed/added to keep the number of changes low... No biggie, I'll fix the indentation of the file in a separate commit.